examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,86 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mm = Matrix([[m1,  0],
+             [0,  m2]])
+f = Matrix([-(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+            k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+mm_full = Matrix([[1, 0,  0,  0],
+                  [0, 1,  0,  0],
+                  [0, 0, m1,  0],
+                  [0, 0,  0, m2]])
+f_full = Matrix([u1,
+                 u2,
+                 -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                 k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the rhs of q' = G(q, u, t, r, p). Kinematic equation
+
+G = Matrix([u1, u2])
+
+# Form the rhs of the dynamics equations
+
+RHS = mm_full.inv()*f_full
+
+# Form the remaining input lists for the equations of motion class
+
+coordinates = (x1, x2)
+speeds = (u1, u2)
+
+# Form the kinetic energy to attach to one of the EOM instances as an output
+# equation
+
+KE = 0.5 * (m1*u1**2 + m2*u2**2)
+out_eqns = {"kinetic_energy": KE}
+
+# Initialize the equation of motion class using the three forms accepted by
+# ODEFunctionGenerator
+#    [1] x' = F(x, t, r, p)
+#
+#    [2] M(x, p) x' = F(x, t, r, p)
+#
+#    [3] M(q, p) u' = F(q, u, t, r, p)
+#        q' = G(q, u, t, r, p)
+
+eom1 = eombase.EOM(coordinates, speeds, rhs=RHS, output_eqns=out_eqns)
+eom2 = eombase.EOM(coordinates, speeds, mass_matrix_full=mm_full,
+                   forcing_full=f_full)
```

Comment:
So I guess we need to decide if System is very general or is System made for multibody systems. That was what some of the discussion in pr #345 was about. I personally would like System to be a multibody system so that it is smart enough to do normal things you want to do with a multibody system. But a `MultibodySystem` could be a subclass to a more general `System`. Systems should also tie nicely into pydy.viz. Right now it is a bit clunky, but a System could be aware of the bodies and loads in the system and make a default visualization.